.travis.yml

```diff
@@ -137,8 +137,6 @@ jobs:
         bucket: getsentry-builds
         upload-dir: $TRAVIS_REPO_SLUG/$TRAVIS_COMMIT
         local_dir: build
-        on:
-          tags: true
```

Comment:
According to the [docs](https://docs.travis-ci.com/user/deployment) the branch doesn't have to be specified if its `master`